libtsuba/src/RDG.cpp

```diff
@@ -320,6 +321,14 @@ tsuba::RDG::DoStoreNodeEntityTypeIDArray(
         "node_entity_type_id_array_file_storage is invalid");
   }
 
+  if (!edge_entity_type_id_array_ff &&
```

Comment:
I can't help but notice that the logic of node type array and edge type array is same. We do similar actions, except a couple of method names are different. @GioLomia : Can you please take a shot at refactoring the common code out into a helper function?